extras/common/app/views/catalog/facet.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render Blacklight::System::ModalComponent.new do |component| %>
```

Comment:
Added this file in order to remove the pagination widget from the modal header. The modal had pagination in the header and footer, but the header pagination was hidden using CSS. This caused a problem for focus management and accessibility in general.